src/shared/components/enhancedReactTable/IColumnFormatterProps.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * @author Selcuk Onur Sumer
+ */
+export interface IColumnFormatterProps {
```

Comment:
Typescript has namespaces, which would allow us to group these interfaces and tie them to the MSKTable, or EnhancedMSKTable as the case may be!